configure.ac

```diff
@@ -716,6 +731,9 @@ fi
 if test "x$build_dtls" != "xyes"; then
     AC_MSG_RESULT([      build DTLS support      : "no"])
 fi
+if test "x$have_epoll" = "xyes"; then
+    AC_MSG_RESULT([      build using epoll       : "$with_epoll"])
```

Comment:
The output might be a bit misleading depending on the config option:

```
configure --with-epoll=no
```

results in:

```
      ....
      build using epoll       : "no"
      ....

$ grep EPOLL coap_config.h
/* #undef COAP_EPOLL_SUPPORT */
#define HAVE_SYS_EPOLL_H 1
```

Maybe we can make it a bit more robust?

Reply:
Hmm - HAVE_SYS_EPOLL_H is not used anywhere (apart from the determination of COAP_EPOLL_SUPPORT in configure.ac), and so with COAP_EPOLL_SUPPORT  correctly defined (not set in the case of --with-epoll=no or --without-epoll) I think that this is still robust unless I am missing something here.

HAVE_SYS_EPOLL_H is still true for Linux based systems as that file does exist, even though the user has elected to have libcoap not built with epoll for doing the i/o socket management.

Reply:
I have found out how not to include HAVE_SYS_EPOLL_H in coap_config.h - and have just pushed the change.